Dockerfile

```diff
@@ -4,13 +4,13 @@ MAINTAINER Atlassian Confluence
 # Setup useful environment variables
 ENV CONFLUENCE_HOME     /var/atlassian/application-data/confluence
 ENV CONFLUENCE_INSTALL  /opt/atlassian/confluence
-ENV CONF_VERSION 6.1.4
+ENV CONF_VERSION 6.7.1
 
 LABEL Description="This image is used to start Atlassian Confluence" Vendor="Atlassian" Version="${CONF_VERSION}"
 
 ENV CONFLUENCE_DOWNLOAD_URL http://www.atlassian.com/software/confluence/downloads/binary/atlassian-confluence-${CONF_VERSION}.tar.gz
 
-ENV MYSQL_VERSION 5.1.38
+ENV MYSQL_VERSION 5.1.45
```

Comment:
Do we have to do anything with RDS because of this?

Reply:
I don't think so, it should be compatible - breaking changes to the mysql protocol should be very rare (definitely not in a point release)